### PR TITLE
Conditional routing to commercial / public space location form

### PR DESCRIPTION
**Pull request by el-mapache**
**SHINY NEW PR**
_now with no weird migration merge conflicts_

[Link to ZenHub issue.](https://github.com/18F/crt-portal/issues/199)

## What does this change?

* This PR adds the conditional routing logic and view for lodging a complaint in a commercial or public space

## Screenshots (for front-end PR):
Screen Shot 2020-01-21 at 12 12 28 PM

Screen Shot 2020-01-21 at 12 12 37 PM

### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/model_variables.py [hunk 1]
```diff
@@ -153,17 +153,22 @@
     ('VOT', _('Voting')),
 )
 
-PLACE_CHOICES = (
-    ('home', _('Home, potential home, or services to help with purchasing a home (banks, lenders, or other financial services)')),
-    ('workplace', _('Workplace or potential workplace')),
-    ('school', _('Educational institution (school, university), education program or educational activity (after school program or workshop)')),
+COMMERCIAL_OR_PUBLIC_PLACE_CHOICES = (
     ('place_of_worship', _('Place of worship')),
```
Comment (LindsayYoung) [inline crt_portal/cts_forms/model_variables.py]: [content doc](https://docs.google.com/document/d/1bxSRc9CQOHjQhTaYkHncE4tykUEFeC6RzFtP5IQk0Ic/edit?ts=5de6f69e#) says "Place of worship or about a place of worship"

File: crt_portal/cts_forms/model_variables.py [hunk 2]
```diff
@@ -153,17 +153,22 @@
...
-    ('store', _('Retail/commercial building (store, restaurant, hotel, nightclub, theater, gym, or other commercial space)')),
-    ('public_space', _('Outdoor public space (including car, street, sidewalk, park)')),
-    ('voting', _('Voting location or ballot (including mail-in ballots)')),
-    ('healthcare', _('Healthcare facility (including mental health or long-term care)')),
-    ('incarcerated', _('Prison, jail, or juvenile corrections facility, or while otherwise incarcerated')),
-    ('government_building', _('Another government building (courthouse, DMV, post office)')),
+    ('store', _('Commercial or retail building')),
+    ('healthcare', _('Healthcare facility')),
+    ('financial', _('Financial institution')),
+    ('public_space', _('Public outdoor space')),
```
Comment (LindsayYoung) [inline crt_portal/cts_forms/model_variables.py]: [content doc](https://docs.google.com/document/d/1bxSRc9CQOHjQhTaYkHncE4tykUEFeC6RzFtP5IQk0Ic/edit?ts=5de6f69e#) says "Public space"

**Comment by LindsayYoung** [comment]
This doesn't seem to have the other option text field.

File: crt_portal/cts_forms/model_variables.py (hunk 2 above)
Comment (el-mapache) [reply crt_portal/cts_forms/model_variables.py]: This must have been updated in the last couple of days, definitely checked there. Thanks for double checking

File: crt_portal/cts_forms/forms.py [hunk 3]
```diff
@@ -344,6 +345,28 @@ def __init__(self, *args, **kwargs):
...
+        self.name = 'CommericalPublicLocation'
+
+        self.fields['commercial_or_public_place'] = TypedChoiceField(
+            choices=COMMERCIAL_OR_PUBLIC_PLACE_CHOICES,
+            empty_value=None,
+            widget=UsaRadioSelect,
+            required=True,
+            error_messages={
+                'required': _('Please select the type of location. If none of these apply to your situation, please select "Other".')
+            }
```
[the remark this answers is not in the document]
Comment (el-mapache) [reply crt_portal/cts_forms/forms.py]: ah, missed carrying that over from the previous PR. thanks 😄

**Comment by el-mapache** [comment]
@LindsayYoung Everything _should_ be fixed up now 🤞

File: crt_portal/cts_forms/model_variables.py (hunk 2 above)
Comment (LindsayYoung) [reply crt_portal/cts_forms/model_variables.py]: No worries, the content does change

**Pull request merged**